guide/adoc/portfile-keywords.adoc

```diff
@@ -219,4 +219,21 @@ license_noconflict openssl
 [source]
 ----
 license_noconflict readline gdbm
+----
+use_xcode::
+Available in 2.6.0 and later. By default, it is assumed on ``macosx`` that ports will not need tools from Xcode.app unless (1) Command Line Tools aren't installed, (2) you are on an old version of Mac OS X that does not support the xcode-select mechanism, or (3) the port uses build.type xcode or is in the xcode PortGroup.
```

Comment:
You should probably put `build.type xcode` and the `xcode` PortGroup in backticks to make them fixed-width display.

I'm also not sure all our users will be aware that `macosx` is a possible value for our `platform` option, so I'd rather change this into "it is assumed on macOS, that..."

Reply:
Done